examples/node/MarketDataSubscription_LongPoll.js

```diff
@@ -1,28 +1,35 @@
 var fs = require('fs');
 var util = require('util');
 var request = require("request");
+var cmdline = require("./cmdline");
 
-var host = process.argv[2] || 'http-api-host';
-var port = 3000;
-var url = 'https://' + host + ':' + port;
-var NUM_REQUEST = 10;   // Number of long-poll requests before unsubscribe
+var NUM_REQUEST = 10; // Number of long-poll requests before unsubscribe.
 var counter = 0;
-var opt = {
-    cert: fs.readFileSync('client.crt'),
-    key: fs.readFileSync('client.key'),
-    ca: fs.readFileSync('bloomberg.crt')
-};
+
+function url()
```

Comment:
But if we go with the way I proposed with `cmdline.js`, the usage will be printed when we source the module in(line 4), so it should be safe to evaluate the variable here.